src/components/table/ExportXlsx.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Download } from 'react-feather';
+import Tooltip from 'components/Tooltip';
+import { useTranslation } from 'react-i18next';
+import Api from 'api';
+import { EXPORT_PEP_XLSX_LIMIT } from 'const/const';
+import YesNoModal from 'components/modals/YesNoModal';
+import toast from 'utils/toast';
+
+
+const ExportXlsx = (props) => {
+  const { t } = useTranslation();
+  const { exportUrl, params, count } = props;
+  const ExportXlsxModalRef = useRef();
+  const detail = 'detail';
```

Comment:
what is this? `const detail = 'detail';`

Reply:
deleted